Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/submission/fragment/SubmissionTestResultFragment.kt

```diff
@@ -32,6 +32,7 @@ class SubmissionTestResultFragment : Fragment(R.layout.fragment_submission_test_
     private val binding: FragmentSubmissionTestResultBinding by viewBindingLazy()
 
     private var skipInitialTestResultRefresh = false
+    private var skipSymptomSubmission = false
```

Comment:
IMHO this should rather be a paramter for continueIfTracingEnabled() --> continueIfTracingEnabled(skipSymptomSubmission = false/true) instead of a predefined value

Reply:
I agree this is more elegant. I will adapt this now.